nautobot/extras/tests/test_jobs.py

```diff
@@ -572,3 +574,73 @@ def test_run(self):
         site_2 = Site.objects.filter(slug="test-site-two")
         self.assertEqual(site_2.count(), 1)
         self.assertEqual(site_2[0].cf["cf1"], "-")
+
+
+class JobHookReceiverTest(TransactionTestCase):
+    """
+    Test job hook receiver.
+    """
+
+    def setUp(self):
+        super().setUp()
+
+        # Initialize fake request that will be required to execute Webhooks (in jobs.)
+        self.request = RequestFactory().request(SERVER_NAME="WebRequestContext")
+        self.request.id = uuid.uuid4()
+        self.request.user = self.user
+
+        # generate an ObjectChange by creating a new site
+        with web_request_context(self.user):
+            site = Site(name="Test Site 1")
+            site.save()
```

Comment:
Can we use `change_logging` instead of `web_request_context` now that your previous PR is in? `web_request_context` should be considered deprecated now, right?

Reply:
I was thinking it might be nice to have context managers for all of the change contexts. 

```py
# instead of having to create the change context
change_context = JobChangeContext(request.user, id=request.id, context_detail=job_model.slug)
with change_logging(change_context):
    _run_job()

# the context manager could do it
with job_change_logging(request.user, id=request.id, context_detail=job_model.slug):
    _run_job()
```